monitoring/definitions/containers.go

```diff
@@ -0,0 +1,45 @@
+package definitions
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/enterprise/dev/ci/images"
+	"github.com/sourcegraph/sourcegraph/monitoring/definitions/shared"
+	"github.com/sourcegraph/sourcegraph/monitoring/monitoring"
+)
+
+func Containers() *monitoring.Container {
```

Comment:
```suggestion
func Containers() *monitoring.Dashboard {
```

Let's just rename this in this PR?

Reply:
Multi-service/container paradigms has been a long-standing request from other teams as well, so might as well get the refactor train going :)

Reply:
i'll open another PR to refact this